services/discovery-service/src/main/java/pl/drobinki/api/services/eureka/DiscoveryServiceApplication.java

```diff
@@ -0,0 +1,13 @@
+package pl.drobinki.api.services.eureka;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+
+@SpringBootApplication
+@EnableEurekaServer
+public class DiscoveryServiceApplication {
```

Comment:
Why you are adding anothre spring boot application here? There is already in first commit DrobinkiApiApplication, you build upon it.

Reply:
If you want create different application, then it should have different repository